static/test/javascripts/spec/common/commercial/creatives/branded-component.spec.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'helpers/injector'
+], function(
+    Injector
+) {
+
+    var config = {
+        images: {
+            commercial: { }
+        }
+    };
+
+    return new Injector()
```

Comment:
Because the module tries to access `config.images.commercial.` (https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/creatives/branded-component.js#L25), so need to set a value for it